activerecord/lib/active_record/callbacks.rb

```diff
@@ -91,11 +91,18 @@ module ActiveRecord
   #
   #   class Topic < ActiveRecord::Base
   #     before_destroy :delete_parents
+  #     around_update :audit
   #
   #     private
   #       def delete_parents
   #         self.class.delete_by(parent_id: id)
   #       end
+  #
+  #       def audit
+  #        # do pre update actions/checks
```

Comment:
We prefer to use real scenarios in our documentation. Can you come up with something that would make case in a `around_update`? Maybe log something.

Reply:
Sure! I'll add a real-case example

Reply:
Hope the example is useful to show the use of the callback